src/modules/create-market/components/create-market-form-liquidity/create-market-form-liquidity.jsx

```diff
@@ -45,9 +45,10 @@ export default class CreateMarketLiquidity extends Component {
       orderEstimate: '',
       minPrice: new BigNumber(0),
       maxPrice: new BigNumber(1),
-      selectedOutcome: this.props.newMarket.type === SCALAR ? 0 : '',
       selectedNav: BID,
     }
+    this.state.selectedOutcome = this.props.newMarket.type === SCALAR ? 0 : ''
```

Comment:
This should be `1` for both scalars and binaries.

Reply:
oh, this was existing code. I'll make the change.